Sketch.java

```diff
@@ -16,21 +16,93 @@ public void settings() {
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(135, 206, 235);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
 	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+  //ground
+  fill(0, 128, 0);
+  rect(0, 250, 400, 250);
+  
+  //calling methods  
+  house(25, 130);
+  house(220, 150);
 
-    stroke(255);
-    line(50, 125, 70, 50);  
+  ball(200, 200, 40);
+  ball(25, 350, 30);
+  ball(350, 350, 50);
+  
+  if(counter == 2){
+  System.out.println(middle(400));
+  counter--;
```

Comment:
indent within your if statement.  Also, not the most effective demonstration of calling a method with a return value.  It'd be better to make use of value returned and use it to draw something